scripts/twister

```diff
@@ -608,6 +608,9 @@ structure in the main Zephyr tree: boards/<arch>/<board_name>/""")
         (twister.csv and twister.xml).
         """)
 
+    parser.add_argument("--monitor-serial-later", action="store_true",
+            help="Monitor device serial after running flash process.")
```

Comment:
This does not explain well what is going on here, the option itself is also awkward. Can you please explain the situation when this is needed and why is it needed, i.e. why do we need a special option when it works without the option most of time?

Reply:
btw, I am curious why we need to invoke ser.close() when met a **serial exception "device reports readiness to read but returned no data (device disconnected or multiple access on port?)**", it seems that this exception maybe caused by other reasons, not only serial disconnected.
```
 try:
            serial_line = ser.readline()
            except TypeError:
                pass
            except serial.SerialException:
                ser.close()
                break
```
because I found I could still got all expected results from serial even I did change as below:
```
            except serial.SerialException:
                pass
```

Reply:
in NXP platform, this will introduce another issue i.e. missing leading log.